tests/portal_account/test_forms.py

```diff
@@ -38,3 +38,30 @@ def test_profile_form_required_fields(self, portal_user):
         form_data = {"user": portal_user, "last_name": "new lname"}
         form = PortalProfileForm(user=portal_user, data=form_data)
         assert not form.is_valid()
+
+
+from django.test import TestCase, Client
```

Comment:
We're using pytest-django in this project. Can you use the pytest fixtures instead? See examples in tests/portal_account/test_views.py